Application/EdFi.Ods.AdminApi/Infrastructure/Extensions/AdminModelExtensions.cs

```diff
@@ -23,4 +24,20 @@ public static class AdminModelExtensions
     {
         return application?.Profiles?.FirstOrDefault()?.ProfileName;
     }
+
+    public static IList<Profile>? Profiles(this Application application)
+    {
+        var profiles = new List<Profile>();
+        foreach (var profile in application.Profiles)
+        {
+            profiles.Add(new Profile { ProfileId =  profile.ProfileId });
```

Comment:
```suggestion
            profiles.Add(new Profile { Id =  profile.Id });
```

Are there two different `Profile` classes? I'm wondering why we can't just `return application.Profiles ?? new List<Profile>();`.

Reply:
@stephenfuqua 
True, we have "Profile" type on  EdFi.Admin.DataAccess.Models.  I tried that one first. But it brings back all the Applications, Api Clients model to Swagger schema. So confusing. To keep it simple, I created new model. Thanks